package-lock.json

```diff
@@ -0,0 +1,3572 @@
+{
```

Comment:
I already suspect the answer's "yes", but just to make sure - are all of these *really* necessary?

Reply:
yes

Reply:
well actually as it stands - these are installed as sub dependencies for dev packages. These arent actually used by our code but probably some sub-dependency that is written in es6 as a node script or something